src/live_restore/live_restore_server.c

```diff
@@ -104,27 +108,41 @@ __live_restore_work_queue_drain(WT_SESSION_IMPL *session)
 /*
  * __live_restore_worker_run --
  *     Entry function for a live restore thread. This is called repeatedly from the thread group
- *     code so it does not need to loop itself.
+ *     code so it does not need to loop itself. Each run will fill the holes for a single file.
  */
 static int
 __live_restore_worker_run(WT_SESSION_IMPL *session, WT_THREAD *ctx)
 {
     WT_DECL_RET;
     WT_LIVE_RESTORE_SERVER *server = S2C(session)->live_restore_server;
+    uint64_t time_diff_ms;
 
     __wt_spin_lock(session, &server->queue_lock);
     if (TAILQ_EMPTY(&server->work_queue)) {
         /* Stop our thread from running. This will call the stop_func and trigger state cleanup. */
         F_CLR(ctx, WT_THREAD_RUN);
-        __wt_verbose_debug1(session, WT_VERB_FILEOPS, "%s", "Live restore worker terminating");
+        __wt_verbose_debug1(session, WT_VERB_LIVE_RESTORE, "%s", "Live restore worker terminating");
         __wt_spin_unlock(session, &server->queue_lock);
         return (0);
     }
     WT_LIVE_RESTORE_WORK_ITEM *work_item = TAILQ_FIRST(&server->work_queue);
     WT_ASSERT(session, work_item != NULL);
     TAILQ_REMOVE(&server->work_queue, work_item, q);
     __wt_verbose_debug2(
-      session, WT_VERB_FILEOPS, "Live restore worker taking queue item: %s", work_item->uri);
+      session, WT_VERB_LIVE_RESTORE, "Live restore worker taking queue item: %s", work_item->uri);
+    WT_STAT_CONN_SET(
+      session, live_restore_queue_length, __wt_atomic_sub64(&server->work_items_remaining, 1));
+    __wt_timer_evaluate_ms(session, &server->msg_timer, &time_diff_ms);
+
+    /* Print out a progress message periodically. */
+    if ((time_diff_ms / (WT_THOUSAND * WT_PROGRESS_MSG_PERIOD)) > server->msg_count) {
+        __wt_verbose(session, WT_VERB_LIVE_RESTORE_PROGRESS,
+          "Live restore has been running for %" PRIu64 " milliseconds and has processed %" PRIu64
+          " file of %" PRIu64,
+          time_diff_ms, server->work_items_remaining, server->work_count);
```

Comment:
This trace counts down so users will see
```
Live restore has been running for 20000 milliseconds and has processed 100 file of 100
Live restore has been running for 40000 milliseconds and has processed 75 file of 100
Live restore has been running for 60000 milliseconds and has processed 50 file of 100
```
I think we want two changes here:
1. `server->work_items_remaining` -> `server->work_count - server->work_items_remaining`
2. `processed %" PRIu64 " file of %" PRIu64` ->  `processed file %" PRIu64 " of %" PRIu64`

Reply:
I think I have addressed this. Weirdly github wasn't showing me this comment earlier today. The result was changing the wording to be `Live restore has been running for 40000 milliseconds and has 75 files of 100 left to process`.

Reply:
@ajmorton and @luke-pearson given the message period is 20 seconds, what do you think of changing it from `40000` milliseconds to `40 seconds`? The current millisecond message is simply based off the RTS progress message but seems too fine-grained for this use. RTS should probably change too but that is outside the scope of this ticket

Reply:
I have decided to go with that change. We don't need millisecond granularity.